src/app/components/list/ItemLabel.scss

```diff
@@ -0,0 +1,35 @@
+.pm_labels {
```

Comment:
I don't like this part:

- We should add a new class instead of using existing one like `pm_label`
- We should never create big selector but prefer to add a new class on the element. Don't do `.pm_label a` but `.my-custom-a`

Reply:
Check with @nico3333fr for good practice.

Reply:
You're right, I should not have used this class. Especially since it refers to html labels and not message labels.

Yet this CSS code was meant to be temporary waiting for @nico3333fr to do it correctly.

Reply:
No problem, this is the old styles, I update them (in my todo list) :)

Reply:
Fixed via https://github.com/ProtonMail/proton-mail/pull/50/commits/06de11ac3ccf5d8a6036eea30e701c2d7a489b51